frontend/src/components/Navbar.tsx

```diff
@@ -207,8 +207,7 @@ function CourseTableNavbar({
                       onClick={() => {
                         posthog.capture('login');
 
-                        window.location.href =
-                          '/legacy_api/index.php?forcelogin=1';
+                        window.location.href = '/api/auth/cas';
```

Comment:
Would be good to always specify the redirect URL instead of relying on the default (here and in other tsx files)